scipy-1.0/paper.tex

```diff
@@ -877,30 +873,28 @@ \subsubsection*{Sparse matrices}
 concerted effort to bring the \texttt{scipy.sparse} API into line with the
 equivalent NumPy API where possible.
 
+\subsubsection*{cKDTree}
+
+\input{ckdtree}
+
 \subsection*{Unified bindings to compiled code}
 
 \subsubsection*{LowLevelCallable}
 
 As of SciPy version 0.19, it is possible for users to wrap low-level functions
-in a \texttt{scipy.LowLevelCallable()} object that reduces the overhead for
-calling compiled C functions directly from Python.  Supported low-level
-functions include \texttt{PyCapsule} objects, ctypes function pointers, and
-cffi function pointers. The low level function signature must be consistent
```

Comment:
It's unclear to me whether/why the details about the function signature are important to include in the paper. The bit about "low level function signature must be consistent with the expectations of the routine it is passed to" seems redundant with "as long as the function call signature matches the specifications". I don't think we need to specify that "The C code may be generated using numba or Cython" as there is no reason for the reader to suspect that the source of the C code matters. The `scipy.ndimage.generic\_filter` example doesn't strengthen the text because the reader is probably unfamiliar with `scipy.ndimage.generic\_filter`.
I'd be happy to help with revisions, though, if this part is really important.

Reply:
`numba` and `Cython` are major open source projects that provide useful ways to interface with Python for preformance reasons, so the interop seems relevant to me.

Some cut down is probably fine as you say.

Reply:
How about 
"As of SciPy version 0.19, it is possible for users to wrap low-level functions
in a \texttt{scipy.LowLevelCallable} object that reduces the overhead of
calling compiled C functions, such as those generated using \texttt{numba} 
or Cython, directly from Python."
?